dev-tools/packaging/templates/docker/Dockerfile.tmpl

```diff
@@ -152,7 +152,7 @@ RUN cd /usr/share/heartbeat/.node \
 RUN chown -R {{ .user }} $NODE_PATH
 USER {{ .user }}
 # If this fails dump the NPM logs
-RUN npm i -g --loglevel verbose --engine-strict @elastic/synthetics@stack_release || sh -c 'tail -n +1 /root/.npm/_logs/* && exit 1'
+RUN npm i -g --loglevel verbose --production --engine-strict @elastic/synthetics@stack_release || sh -c 'tail -n +1 /root/.npm/_logs/* && exit 1'
```

Comment:
Just for my own edification: is this change related to the upgrade, or just an incidental improvement?

Reply:
@axw incidental improvement to shave off some install time

Reply:
thanks!